bioagents/msa/msa_module.py

```diff
@@ -115,5 +280,33 @@ def _matching(self, stmt, residue, position, action, polarity):
         return matching_residues
 
 
+def _make_sbgn(stmts):
+    sa = SBGNAssembler()
+    sa.add_statements(stmts)
+    sa.make_model()
+    sbgn_str = sa.print_model()
+    logger.info(sbgn_str)
+    return sbgn_str
+
+
+def _make_diagrams(stmts):
+    sbgn = _make_sbgn(stmts)
+    # rxn = draw_reaction_network(pysb_model, model_id)
+    # cm = draw_contact_map(pysb_model, model_id)
+    # im = draw_influence_map(pysb_model, model_id)
+    # diagrams = {'reactionnetwork': rxn, 'contactmap': cm,
+    #             'influencemap': im, 'sbgn': sbgn}
+    diagrams = {'sbgn': sbgn.decode('utf-8')}
+    return diagrams
+
+
+def _assemble_pysb(stmts):
```

Comment:
This block isn't (and probably won't be) used here so could be removed

Reply:
def _assemble_pysb(stmts)

Reply:
Oh, Ok. So we won't try to get the pysb assembly working for these sets of statements?

Reply:
No because reaction network generation could easily become intractable or even infinite in some situations, that's why using the SBGNAssembler in this context is better.